jwst/outlier_detection/outlier_detection_step.py

```diff
@@ -64,11 +64,15 @@ class OutlierDetectionStep(Step):
         scale_detection = boolean(default=False)
         search_output_file = boolean(default=False)
         allowed_memory = float(default=None)  # Fraction of memory to use for the combined image.
+        in_memory = boolean(default=False)
+        output_use_index = boolean(default=False)
+        output_use_model = boolean(default=False)
```

Comment:
I do not believe these three new step parameters are used anywhere in the step. If that is correct, these parameters should be removed.
```suggestion
```

Reply:
They may be standard params that are defined in the standard `Step` class and hence you're right that they're not needed here (already available in the step object created for the process).